components/plan-build-ps1/bin/hab-plan-build.ps1

```diff
@@ -287,10 +287,23 @@ function Invoke-Begin {
   Invoke-DefaultBegin
 }
 
-# Default implementation for the `Invoke-Begin` phase.
+# Default implementation for the `Invoke-Before` phase.
```

Comment:
Oh sure. I'm was referring to the comment just above the default invoke-begin. It looked like the result of an over zealous search and replace.